frontend/packages/dev-console/src/components/helm/HelmReleaseResourceTableHeader.tsx

```diff
@@ -2,9 +2,9 @@ import { sortable } from '@patternfly/react-table';
 
 export const tableColumnClasses = {
   name: 'col-lg-4 col-md-4 col-sm-4 col-xs-6',
-  kind: 'col-lg-2 col-md-2 col-sm-4 col-xs-6',
```

Comment:
these should be indexed, instead of names, so that you do not need to update each time there is a name change